lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -117,6 +117,13 @@ class ReportUIFeatures {
       this._document.addEventListener('scroll', this._updateStickyHeaderOnScroll);
       window.addEventListener('resize', this._updateStickyHeaderOnScroll);
     }
+
+    // Show the metric descriptions by default when there is an error.
+    if (report.audits.metrics.errorMessage) {
```

Comment:
worth noting that TTI, speed index, and FCPUI will just be undefined and not throw an errorMessage onto metrics so this will only handle it for fcp/fmp/eil/traceOfTab errors

but that seems to be the case we want to auto-expand anyhow, when all of them fail basically?

Reply:
My intention was to expand if there are any "Error"s in the metric table. I assumed audits.metrics.errorMessage would be that, but that's probably a bad assumption...

Reply:
ah you'll probably want to switch to `report.audits['first-contentful-paint'].errorMessage || report.audits.interactive.errorMessage || ...` then :/

Reply:
figured it out